draftjs_exporter/engines/string.py

```diff
@@ -0,0 +1,107 @@
+from __future__ import absolute_import, unicode_literals
+
+from draftjs_exporter.engines.base import DOMEngine
+
+try:
+    # Python 3.2 and above.
+    from html import escape
+except ImportError:
+    import cgi
+
+    def escape(s):
+        # Force quote escaping in Python 2.7.
+        return cgi.escape(s, quote=True).replace('\'', '&#x27;')
+
+# http://w3c.github.io/html/single-page.html#void-elements
+# https://github.com/html5lib/html5lib-python/blob/0cae52b2073e3f2220db93a7650901f2200f2a13/html5lib/constants.py#L560
+VOID_ELEMENTS = [
+    'area',
+    'base',
+    'br',
+    'col',
+    'embed',
+    'hr',
+    'img',
+    'input',
+    'link',
+    'meta',
+    'param',
+    'source',
+    'track',
+    'wbr',
+]
+
+
+class DOM_STRING(DOMEngine):
```

Comment:
Any particular reason for this class name instead of DOMString?

Reply:
Ah, no, that's just me being silly. I probably tried to replicate the somewhat questionable nomenclature of other engines (`DOM_HTML5LIB` and `DOM_LXML`). Going for `DOMString` instead.